pyne/xs/cache.py

```diff
@@ -112,7 +113,8 @@ def __getitem__(self, key):
                 else:
                     raise KeyError
         # Return the value requested
-        return self._cache[key]
+        scaler = self._xs_scalers.get(kw['nuc'], 1)
```

Comment:
I would push this logic farther up so that the scaled value is stored in the cache, rather than recomputing it each time.  Also, I would make it so the default is None, and then the value is only scaled if not None.  That prevents extra floating point ops that aren't needed.